src/main/java/io/vertx/core/http/impl/HttpServerImpl.java

```diff
@@ -112,14 +106,12 @@
   private static final boolean DISABLE_WEBSOCKETS = Boolean.getBoolean(DISABLE_WEBSOCKETS_PROP_NAME);
   private static final String DISABLE_H2C_PROP_NAME = "vertx.disableH2c";
   private final boolean DISABLE_H2C = Boolean.getBoolean(DISABLE_H2C_PROP_NAME);
-  private static final String[] H2C_HANDLERS_TO_REMOVE = { "idle", "flashpolicy", "deflater", "chunkwriter" };
```

Comment:
We will not remove those anymore now?

Reply:
they are actually all removed during the upgrade now in `Http2UpgradeHandler`

```
              ChannelPipeline pipeline = ctx.pipeline();
              Iterator<Map.Entry<String, ChannelHandler>> iterator = pipeline.iterator();
              while (iterator.hasNext()) {
                Map.Entry<String, ChannelHandler> handler = iterator.next();
                if (handler.getValue() instanceof Http2ConnectionHandler) {
                  // Continue
                } else {
                  iterator.remove();
                }
              }
```